React/React-Core.podspec

```diff
@@ -28,16 +28,28 @@ Pod::Spec.new do |s|
   s.homepage               = "http://facebook.github.io/react-native/"
   s.license                = package["license"]
   s.author                 = "Facebook, Inc. and its affiliates"
-  s.platforms              = { :ios => "9.0", :tvos => "9.2" }
+  s.platforms              = { :ios => "9.0", :tvos => "9.2", :osx => "10.14" }
   s.source                 = source
   s.source_files           = "**/*.{c,h,m,mm,S,cpp}"
   s.exclude_files          = "DevSupport/**/*",
                              "Fabric/**/*",
                              "Inspector/**/*"
   s.ios.exclude_files      = "**/RCTTV*.*",
-# Microsoft CocoaPod Support
-														 "**/MacOS/*"
-# Microsoft CocoaPod Support
+# Microsoft CocoaPods Support
```

Comment:
Can we change instances of this tag/comment to whatever tag you're using elsewhere? Can probably get rid of the word "Microsoft" at least